test/apiv2/30-volumes.at

```diff
@@ -3,12 +3,75 @@
 # volume-related tests
 #
 
-#
-# FIXME: endpoints seem to be unimplemented, return 404
-#
-if false; then
-t GET  libpod/volumes/json            200 null
-t POST libpod/volumes/create name=foo 201
-fi
+## create volume
+
+voluempath=$(curl -s http://$HOST:$PORT/v1.40/libpod/info | jq -r ".store.volumePath")
```

Comment:
Did you mean to make this "voluempath" instead of "volumepath"?

Reply:
It's a typo,  I have corrected it. Thanks.